### PR TITLE
Fix authors not showing

### PR DESCRIPTION
**Pull request by samuveth**
Fixes https://github.com/snapshot-labs/snapshot/issues/1597

Changes proposed in this pull request:
- Don't await getSpaceUri before formatting space

### CODE REVIEW

**Comment by mktcode** [comment]
Reset buttons still doesn't work for me.

**Comment by samuveth** [comment]
> From the issue:
> 
> The reset button in the settings will also not work properly because of this.
> 
> Reset button still doesn't work for me. Most fields can be reset once but not again. Changing author/admin addresses doesn't seem to get reset at all.

Oh yes, this is a bigger issue I need to solve with the way our Textarea components work. Couldn't figure it out yet, so I will open an issue for it. Then we can merge this fix first.

**Comment by midgerate** [comment]
@samuveth still not working

https://www.loom.com/share/9a1f5a68da364f85b1a780d354e11e59

**Comment by samuveth** [comment]
Sorry for that, fixed and did more tests this time :)

**Comment by samuveth** [comment]
Converted to draft will make some more changes

**Comment by samuveth** [comment]
How about this?
Feb-04-2022 17-51-25

**Comment by samuveth** [comment]
How about like this?
Feb-04-2022 18-29-37

File: src/views/SpaceSettings.vue [hunk 1]
```diff
@@ -155,14 +155,22 @@ function inputError(field) {
...
+    if (currentSettings.value)
+      return (form.value = clone(currentSettings.value));
+    form.value = {
+      strategies: [],
+      categories: [],
+      plugins: {},
+      filters: {}
+    };
+  } finally {
+    loaded.value = false;
```
Comment (midgerate) [inline src/views/SpaceSettings.vue:169]: Looks like an antipattern, if there's a reason to do something like this, its best to write a comment

Comment (mktcode) [reply src/views/SpaceSettings.vue:169]: As far as I understood it's a workaround for an issue with the textarea component's state. A comment would be nice though for sure. :D

Comment (samuveth) [reply src/views/SpaceSettings.vue:170]: Agree. Added a comment thanks

Comment (mktcode) [reply src/views/SpaceSettings.vue:170]: @samuveth Does [this](https://github.com/snapshot-labs/snapshot/pull/1647) maybe help to avoid this workaround?

Comment (samuveth) [reply src/views/SpaceSettings.vue:170]: Ok yes let's do that. Looks good btw haven't tested it yet

**Review by midgerate — APPROVED** [review]

**Pull request merged**